R/hp_datatree2.R

```diff
@@ -60,25 +32,32 @@ dt_collect <- function(x) {
 #' @export
 hp_datatree <- function(.data, title = "Hierplane", attributes = NULL, link = "to",
                         styles = NULL) {
+
   requireNamespaceQuietStop("data.tree")
+
+
+  # clean up the slashes
+  .data$Set(name = gsub("[/]", "~~~placeholder~~~", .data$Get("name")))
+
+
   root <- dt_root_df(.data)
   df <- vctrs::vec_rbind(dt_collect(.data), root)
 
+
   # should only expect a single NA in the link column, this is NA comes
   # from binding the root dataframe to the tree. If there are multiple
   # NAs, then the there are missing links
-  missing_links <- "TRUE" %in% names(table(is.na(df[[link]])))
```

Comment:
do you want to talk about this lol

Reply:
Haha, yes... I forget about summing logicals, it still feels unnatural to me tho.